FilePersistence/src/version_control/merge/ListMergeComponentV2.h

```diff
@@ -40,6 +41,13 @@ class FILEPERSISTENCE_API ListMergeComponentV2 : public MergePipelineComponent
 		static bool isList(const QString& type);
 		static bool isUnorderedList(const QString& type);
 		static bool isOrderedList(const QString& type);
+
+		QList<Model::NodeIdType> computeListsToMerge(MergeData& mergeData);
+		QMultiHash<Model::NodeIdType, QPair<QString, QString>>
+			getAdjustedIndices(Model::NodeIdType list, MergeData& mergeData);
+		void adjustCG(QMultiHash<Model::NodeIdType, QPair<QString, QString>> map, MergeData& mergeData);
+		QList<Model::NodeIdType> nodeListToSortedIdList(const QList<GenericNode*>& list);
```

Comment:
Which one is this function exactly? I find the fact that it takes generic nodes as inputs but returns IDs a bit confusing. Where do we call this?